Jenkinsfile

```diff
@@ -66,7 +66,6 @@ timestamps {
                     'admin',
                     'api',
                     'authorization-service',
-                    'discover-publish',
```

Comment:
does [this switch case](https://github.com/Pennsieve/pennsieve-api/pull/284/files#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8R138-R141) need to be removed as well?